internal/graph/check.go

```diff
@@ -329,16 +352,29 @@ func (c *LocalChecker) dispatch(ctx context.Context, req *ResolveCheckRequest) C
 	}
 }
 
-func (c *LocalChecker) ResolveCheck(
+func (c *LocalChecker) Execute(
```

Comment:
Leave as is please. See https://github.com/openfga/openfga/pull/866/files#r1262629852

The composition of this was purposeful as well. It should dispatch back to itself, because that's what would have to happen if you invoked the dispatch remotely.